CHANGELOG.md

```diff
@@ -81,6 +81,7 @@
 - PR #1845 Add cudf::drop_duplicates, DataFrame.drop_duplicates
 - PR #1652 Added `Series.where()` feature
 - PR #2074 Java Aggregates, logical ops, and better RMM support
+- PR #2179 Added quantiles
```

Comment:
this is listed in 0.8, we should move above to 0.9.

Reply:
good catch